crates/optimism/rpc/src/sequencer.rs

```diff
@@ -104,16 +83,8 @@ impl SequencerClient {
         condition: TransactionConditional,
     ) -> Result<(), SequencerClientError> {
         let params = json!([format!("0x{}", hex::encode(tx)), condition]);
-        let body =
-            self.request_body("eth_sendRawTransactionConditional", params).map_err(|_| {
-                warn!(
-                    target: "rpc::eth",
-                    "Failed to serialize transaction for forwarding to sequencer"
-                );
-                SequencerClientError::InvalidSequencerTransaction
-            })?;
 
-        self.post_request(body).await.inspect_err(|err| {
+        self.send_rpc_call("eth_sendRawTransaction", params).await.inspect_err(|err| {
```

Comment:
is it intentional that this now calls `eth_sendRawTransaction` instead of `eth_sendRawTransactionConditional`?

Reply:
my bad, it was not intentional , fixed it . Restored the call to `eth_sendRawTransactionConditional`